shared/AppInsightsCore/src/JavaScriptSDK/CookieMgr.ts

```diff
@@ -288,6 +338,25 @@ export function createCookieMgr(rootConfig?: IConfiguration, logger?: IDiagnosti
             if (_isMgrEnabled(cookieMgr)) {
                 // Only remove the cookie if the manager and cookie support has not been disabled
                 result = cookieMgr.purge(name, path);
+            } else {
+                // Cache the deletion operation when cookies are disabled
+                // Format the deletion cookie string to use when cookies are re-enabled
+                let values = {
+                    [STR_PATH]: path ? path : "/",
+                    [strExpires]: "Thu, 01 Jan 1970 00:00:01 GMT"
+                };
+
+                if (!isIE()) {
+                    // Set max age to expire now
+                    values["max-age"] = "0";
+                }
+
+                _pendingCookies[name] = {
+                    operation: 'delete',
```

Comment:
Lets change the state to "purge" to match the operation being called on the manager